sdk/appconfiguration/app-configuration/test/README.md

```diff
@@ -0,0 +1,10 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+The environment variables you will need to properly assign to run the live test of this project are the following:
```

Comment:
We should have a line about what Azure resource is needed to run the tests.

```suggestion
To run the live tests, you need to have an [Azure App Configuration account](link to some docs that talk about how to create this account) and set the below environment variables:
```

Reply:
Use similar format for other packages as well

Reply:
on it